tsl/src/continuous_aggs/materialize.c

```diff
@@ -272,6 +272,11 @@ spi_delete_materializations(SchemaAndName materialization_table, const NameData
 
 	if (res < 0)
 		elog(ERROR, "could not delete old values from materialization table");
+	else
+		elog(LOG,
+			 "deleted %lu row%s from materialization table",
```

Comment:
Since multiple jobs could be executed, it would be good to log the table name. This would also allow the user to search for specific materialization table updates.